.pre-commit-config.yaml

```diff
@@ -106,31 +112,27 @@ repos:
     rev: 3.1.8
     hooks:
       - id: setuptools-odoo-make-default
+      - id: setuptools-odoo-get-requirements
+        args:
+          - --output
+          - requirements.txt
+          - --header
+          - "# generated from manifests external_dependencies"
   - repo: https://github.com/PyCQA/flake8
     rev: 3.9.2
     hooks:
       - id: flake8
-        name: flake8 except __init__.py
-        exclude: /__init__\.py$
-        additional_dependencies: ["flake8-bugbear==20.1.4"]
-      - id: flake8
-        name: flake8 only __init__.py
-        args: ["--extend-ignore=F401"] # ignore unused imports in __init__.py
```

Comment:
I think this one is important, specially for the now included migration tests.

Reply:
Hi. Thanks for your review. 

1) Not sure to understand. Do you mean you want to keep : 
```
      - id: flake8
        name: flake8 except __init__.py
        exclude: /__init__\.py$
        additional_dependencies: ["flake8-bugbear==20.1.4"]
```

or 
```
      - id: flake8
        name: flake8 only __init__.py
        args: ["--extend-ignore=F401"] # ignore unused imports in __init__.py
        files: /__init__\.py$
        additional_dependencies: ["flake8-bugbear==20.1.4"]
```
or both ?

2) In other OCA repo, that line (--extend-ignore=F401) is not present. Are you sure it is required ? 
https://github.com/OCA/server-tools/blob/16.0/.pre-commit-config.yaml#L124